WooCommerce/Classes/ViewRelated/Orders/Order Creation/Synchronizer/RemoteOrderSynchronizer.swift

```diff
@@ -56,6 +56,53 @@ final class RemoteOrderSynchronizer: OrderSynchronizer {
     ///
     private let localIDStore = LocalIDStore()
 
+    // MARK: Private publishers for significant order input
```

Comment:
I think it is worth having a small explanation on why these extra inputs are needed! What do you think?